packages/cells/src/widget.ts

```diff
@@ -479,6 +479,9 @@ class CodeCell extends Cell {
     let contentFactory = this.contentFactory;
     let model = this.model;
 
+    // Code cells should not wrap lines.
+    this.editor.wordWrap = false;
```

Comment:
I think this is fine for now, but we should think more about localizing defaults to consistent places.